controllers/send-widget.controller.es6

```diff
@@ -359,9 +364,23 @@ export default class SendWidgetController {
           .addMemo(memo)
           .build();
 
-        transaction.sign(Keypair.fromSeed(this.session.getSecret()));
-
-        return this.Server.submitTransaction(transaction);
+        if (this.useLedger) {
+          let ledgerApi = new StellarLedger.Api(new StellarLedger.comm(120));
+          return ledgerApi.signTx_async(this.bip32Path, transaction).then(result => {
```

Comment:
What do you think about changing this to `signTxAsync`?

Reply:
It's a better name. I'll add it to the things todo for the next release.

Reply:
As in https://github.com/stellar/account-viewer/pull/43#discussion_r151986414, if it's idiomatic to Ledger, let's leave it.